### PR TITLE
Update npm deps

### PR DESCRIPTION
**Pull request by prymitive**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.9.1-alpine3.6 as unsee-builder
+FROM golang:1.9.2-alpine3.6 as unsee-builder
 COPY . /go/src/github.com/cloudflare/unsee
 
 ARG VERSION
```
Comment (terinjokes) [inline Dockerfile]: nit: it might be better to use a nodejs image here, rather than installing Node.js into a Go build image.

Comment (prymitive) [reply Dockerfile]: but then I need to install go into nodejs image

Comment (terinjokes) [reply Dockerfile]: You can have multiple stages.

Comment (prymitive) [reply Dockerfile]: That would be unrelated to version bumps here, feel free to open a ticket for that

**Pull request merged**